backend/api_v2/management/commands/delete_org_rate_limit.py

```
@@ -0,0 +1,103 @@
+            self.stdout.write(
+                f'\nCurrent usage: {usage["org_count"]}/{default_limit} concurrent requests'
+            )
+
+            if usage["org_count"] > default_limit:
+                self.stdout.write(
+                    self.style.ERROR(
+                        "WARNING: Current usage exceeds default limit! "
+                        "New requests will be rate limited until usage drops."
+                    )
+                )
+        except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)